core-blocks/button/index.js

```diff
@@ -142,7 +130,7 @@ export const settings = {
 		},
 
 		save( { attributes } ) {
-			const { url, text, title, align, color, textColor } = attributes;
+			const { url, text, title, color, textColor } = attributes;
```

Comment:
I think we should include explicitly `align` handling in both `deprecated` definitions. Have you tested it with an old version of HTML?

Reply:
Addressed in [this](https://github.com/WordPress/gutenberg/pull/7100/commits/9374ed318ead245d1b3ba974307c1c7a62bc7096) commit. Thank you ❤️